library.c

```diff
@@ -2150,6 +2150,8 @@ redis_sock_create(char *host, int host_len, int port,
     redis_sock->host = zend_string_init(host, host_len, 0);
     redis_sock->status = REDIS_SOCK_STATUS_DISCONNECTED;
     redis_sock->retry_interval = retry_interval * 1000;
+    redis_sock->max_retries = 10;
+    redis_sock->backoff = make_default_backoff(retry_interval);
```

Comment:
I prefer `redis_initialize_backoff(&redis_sock->backoff, retry_interval)`

Reply:
Both strategies are well defined and valid C but I'm agnostic :smile:

Reply:
I switched it to the proposed approach